src/main/java/bio/terra/app/controller/SearchApiController.java

```diff
@@ -95,27 +95,28 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
 
     @Override
     public ResponseEntity<SearchQueryResultModel> querySearchIndices(
-        @Valid @RequestBody SearchQueryRequest searchQueryRequest,
-        @RequestParam(value = "offset", required = false, defaultValue = "0") Integer offset,
-        @RequestParam(value = "limit", required = false, defaultValue = "1000") Integer limit
-    ) {
-        List<String> accessibleIds =
-                iamService.listAuthorizedResources(getAuthenticatedInfo(), IamResourceType.DATASNAPSHOT)
-                        .stream()
-                        .map(UUID::toString)
-                        .collect(Collectors.toList());
+        SearchQueryRequest searchQueryRequest,
+        Integer offset,
+        Integer limit) {
+
+        List<UUID> accessibleIds =
+            iamService.listAuthorizedResources(getAuthenticatedInfo(), IamResourceType.DATASNAPSHOT).subList(0, 10);
```

Comment:
whoops, this was added for testing
```suggestion
            iamService.listAuthorizedResources(getAuthenticatedInfo(), IamResourceType.DATASNAPSHOT);
```